sandbox.go

```diff
@@ -71,9 +71,21 @@ func (s *sandbox) execute() (string, error) {
 
 	dockerCommand := s.options.path + "/DockerTimeout.sh"
 
-	args := []string{fmt.Sprintf("%s", s.options.timeout), "-u", "mysql", "-i", "-t", "--volume=" + s.options.folder + ":/usercode", s.options.image, "/usercode/script.sh", compiler, filename, optionalExecutable, flags}
-
-	done := make(chan error)
+	args := []string{
+		fmt.Sprintf("%s", s.options.timeout),
+		"-u",
+		"mysql",
+		"-i",
+		"-t",
+		"--volume=" + s.options.folder + ":/usercode",
+		s.options.image,
+		"/usercode/script.sh",
+		compiler,
+		filename,
+		optionalExecutable,
+		flags}
+
+	done := make(chan error, 1)
```

Comment:
i'm a tiny bit confused why increasing the buffer size would prevent timeouts from occurring; since it seems like we are creating one `done` chan per sandbox execution, a buffer of zero shouldn't interfere with independent sandboxes.

Reply:
No, this has nothing to do with the actual fix, it's just something I tried early on.

It shouldn't matter, so I'm happy to revert this change.

Reply:
Reverted and rewrote the commit/PR message to better reflect the actual fix.